content/learn/quick-start/getting-started/setup.md

```diff
@@ -344,6 +344,16 @@ rustflags = [
 
 </details>
 
+### Improve Runtime Performance (Optional)
+
+Bevy's dependencies do a lot of trace logging that is not relevant for an end user. 
+To improve your runtime performance, you can add the following to the `[dependencies]` section of your Cargo.toml. 
+It will disable detailed log levels on compile time so that they do not need to be filtered out while your app is running.
+
+```toml
+log = { version = "0.4", features = ["max_level_debug", "release_max_level_warn"] }
```

Comment:
Should it be log version = "*", so that user's don't have to worry about the version?

Reply:
No. It needs to be in sync with Bevy. That said, it's been half a decade since 0.4 released.

Reply:
I think "*" should let it synchronize with any other version in use (such as that used by Bevy). But yeah, no strong feelings here.

Reply:
@alice-i-cecile oh, I thought `*` meant "latest version", but quick googling tells me that's a common misconception. Changing it.

Reply:
[`497f582` (#1503)](https://github.com/bevyengine/bevy-website/pull/1503/commits/497f5825a429bb792e477727c611da4a7e5bfb33)